### PR TITLE
Firefox support + small improvements

### PR DESCRIPTION
**Pull request by honsiorovskyi**
Not sure if these changes make any sense, but I liked the extension and decided to make a FF version for it too. Additionally I did some tiny changes to make it look a bit nicer (well, at least on my screen). I also added Prettier config for code formatting.

### CODE REVIEW

File: firefox/src/manifest.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "New Tab Page Bookmarks",
+  "description": "Displays your bookmarks in multiple columns for quick access from the new tab page",
+  "version": "1.1",
+  "manifest_version": 2,
```
Comment (honsiorovskyi) [inline firefox/src/manifest.json:5]: This is the only change in manifest, FF doesn't like v3, Chrome doesn't like v2 (ノಠ益ಠ)ノ彡┻━┻

File: package.json [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+  "version": "1.1.0",
   "main": "index.js",
   "license": "MIT",
   "author": "",
   "scripts": {
-    "build": "zip -j chrome-extension.zip src/*"
+    "build-chrome": "mkdir -p dist/chrome-extension; cp -r src/* chrome/src/* dist/chrome-extension && zip -j dist/chrome-extension.zip dist/chrome-extension/*",
+    "build-firefox": "mkdir -p dist/firefox-extension; cp -r src/* firefox/src/* dist/firefox-extension && zip -j dist/firefox-extension.zip dist/firefox-extension/*",
+    "clean": "rm -rf dist",
+    "build": "\"$npm_execpath\" build-chrome && \"$npm_execpath\" build-firefox"
```
Comment (honsiorovskyi) [inline package.json:11]: This crazy thing makes is possible to zip it for both browsers and also creates `dist/${browser}-extension` folder that can be loaded unpacked into corresponding browsers.

File: src/index.css [hunk 3]
```diff
@@ -4,17 +4,20 @@
 }
 body {
 	font-size: 13px;
+	font-family: sans-serif;
```
Comment (honsiorovskyi) [inline src/index.css:7]: For some reason, it falls back to `serif` by default in my browser, so I forced it to `sans-serif`.

File: src/index.css [hunk 4]
```diff
@@ -4,17 +4,20 @@
...
 }
 ul#root {
 	display: flex;
 	flex-direction: row;
 	flex-wrap: nowrap;
+	justify-content: space-around;
```
Comment (honsiorovskyi) [inline src/index.css]: Distribute the columns evenly across the screen.

File: src/index.css [hunk 5]
```diff
@@ -4,17 +4,20 @@
...
 	margin-left: 20px;
 }
 ul#root > li {
 	margin-right: 20px;
 }
 h3 {
+	margin-top: 15px;
```
Comment (honsiorovskyi) [inline src/index.css:19]: A bit of space between sub-folders.

File: src/index.css [hunk 6]
```diff
@@ -38,4 +41,5 @@ a > img {
 	margin-right: 4px;
 	width: 16px;
 	height: 16px;
+	vertical-align: text-bottom;
```
Comment (honsiorovskyi) [inline src/index.css:43]: Align text and icon to be on the same level.

File: src/index.js [hunk 7]
```diff
@@ -1,39 +1,47 @@
...
+    const root = document.querySelector('#root')
+    rootNode.children.forEach(renderNode(root))
+  })
 }
 
-chrome.bookmarks.getSubTree(bookmarksBarId, function([ rootNode ]) {
-	const root = document.querySelector('#root')
-	rootNode.children.forEach(renderNode(root))
-})
+const bookmarksBarId = !!chrome.app ? '1' : 'toolbar_____'
```
Comment (honsiorovskyi) [inline src/index.js:46]: Different IDs for the bookmark bars in Chrome & Firefox.

File: src/index.css (hunk 4 above)
Comment (mathio) [reply src/index.css]:
```suggestion
```

Sorry, I dont want the spacing around. The columns are aligned to the left intentionally.

**Review by mathio — APPROVED** [review]
Sorry for neglecting your PR. Approved with 1 minor comment that is non-negotiable.

**Pull request merged**